src/app/projects/project-details/project-details-build/project-details-build.component.html

```diff
@@ -25,11 +20,13 @@
       <td class="col-environment">{{build.environment}}</td>
       <td class="col-status">
         <span [class]="this.getStatusCssClass(build.statusId)">{{build.status}}</span>
-        <p-progressSpinner
-          *ngIf="isBuildRunning(build)"
-          class="status-progress-spinner"
-          strokeWidth="4"
-          ></p-progressSpinner>
+        <p-progressSpinner *ngIf="isBuildRunning(build)" class="status-progress-spinner" strokeWidth="4">
+        </p-progressSpinner>
+      </td>
+      <td class="col-tests">
+        {{build.testResultListSummary.failed | number:'1.0-0'}}<i class="pi pi-times" [ngStyle]="{'color': (build.testResultListSummary.failed) ? 'red' : 'gray'}"></i>
+        {{build.testResultListSummary.skipped | number:'1.0-0'}}<i class="pi pi-circle-on" [ngStyle]="{'color': (build.testResultListSummary.skipped) ? 'orange' : 'gray'}"></i>
+        {{build.testResultListSummary.passed | number:'1.0-0'}}<i class="pi pi-check" [ngStyle]="{'color': (build.testResultListSummary.passed) ? 'green' : 'gray'}"></i>
```

Comment:
nit: Use classes instead of directly changing the styles.

Same in
- `src/app/test-results/test-results-list/test-results-detail-list.component.html`
- `src/app/test-results/test-results-list/test-results-summary-list-item.component.html`

Reply:
Opted to use classes with attributes, using separate classes and `[ngClass]="{ 'class-1': cond, 'class-2': !cond }"` to select felt very ugly and verbose. If you know a better way of doing that that'd be neat.